enrollment-server/src/main/java/com/wultra/app/enrollmentserver/controller/api/InboxController.java

```diff
@@ -160,7 +161,7 @@ public Response readMessage(@RequestBody ObjectRequest<InboxReadRequest> objectR
             PowerAuthSignatureTypes.POSSESSION_KNOWLEDGE,
             PowerAuthSignatureTypes.POSSESSION_KNOWLEDGE_BIOMETRY
     })
-    public Response readAllMessages(@RequestBody ObjectRequest<InboxReadAllRequest> objectRequest, @Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication) throws InboxException {
+    public Response readAllMessages(@Parameter(hidden = true) PowerAuthApiAuthentication apiAuthentication) throws InboxException {
```

Comment:
The same with unused class as for `countUnreadMessages()`

Reply:
dtto, good point.